rest-api-spec/src/main/resources/rest-api-spec/api/ml.put_trained_model_definition_part.json

```
@@ -0,0 +1,38 @@
+{
+  "ml.put_trained_model_definition_part":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/put-trained-model-definition-part.html",
+      "description":"Adds part of a trained model definition"
```

Since we ideally want to generate the docs from these specs in the long run, they should match as much as possible:
```suggestion
      "description":"Creates part of a trained model definition"
```